brig/cmd/brig/commands/project_create.go

```diff
@@ -338,13 +339,13 @@ func projectCreatePrompts(p *brigade.Project) error {
 	}, &doAdvanced, nil); err != nil {
 		return fmt.Errorf(abort, err)
 	} else if doAdvanced {
-		return projectAdvancedPrompts(p)
+		return projectAdvancedPrompts(p, store)
 	}
 	return nil
 }
 
-func projectAdvancedPrompts(p *brigade.Project) error {
-	if err := survey.Ask([]*survey.Question{
+func projectAdvancedPrompts(p *brigade.Project, store *storage.Store) error {
```

Comment:
Why pass pointer to `Store` here?

Reply:
[This article](https://medium.com/@saiyerram/go-interfaces-pointers-4d1d98d5c9c6) explains it pretty well